backends/sqs/naming.go

```diff
@@ -23,8 +26,13 @@ func QueueUrlToQueueId(url string) (*rpc.QueueId, error) {
 	if len(name_tokens) != 2 {
 		return nil, fmt.Errorf("Malformed queue name %v", name)
 	}
+	queueType := rpc.QueueId_Standard
+	if name[len(name)-5:] == ".fifo" {
```

Comment:
not sure if I fully understand the code, but when the name contains .fifo, we consider it as fifo queue, that seems right, but shouldn't we strip the 'fifo' suffix' from the name? Or put it another way, if the name part in a rpc request already contains 'fifo', should we check for that and don't append another .fifo suffix?